### ISSUE TITLE
How to dump muGraph?

### ISSUE THREAD

**Issue by zhaosiying12138** [mirage-project/mirage#145]
Sorry to ask a naive question, but I would like to dump the optimatized muGraph searched by mirage, how can I do that?

**Comment by zhaosiying12138** [comment]
![微信图片_20241121221842](https://github.com/user-attachments/assets/a68e1876-d1e8-47df-bb79-f6f159eb9964)
自问自答，it seems I can run python python/mirage/visualizer.py to get muGraph！Beautiful!

**Comment by jiazhihao** [comment]
Yes, you can use the visualizer to draw the muGraphs. Mirage also supports API to print the generated CUDA program for each muGraph.
